extensions/notion/src/components/forms/CreatePageForm.tsx

```diff
@@ -91,14 +117,25 @@ export function CreatePageForm({ mutate, launchContext, defaults }: CreatePageFo
                 }
               : undefined,
           });
+        } else {
+          await showHUD("Page created ✅", { popToRootType: PopToRootType.Immediate, clearRootSearch: true });
```

Comment:
`showToast` will show an improved HUD instead if Raycast is closed, so there's no need to use `showHUD` here 🙂

Reply:
I will change that

Reply:
@thomaslombart  Changed